images/sources-readme.txt

```diff
@@ -0,0 +1,7 @@
+
+Due to the way this image is built, the source container for the image itself
+does not include the full set of sources for the cli tool binaries included in
+the image, i.e. `cosign`, `gitsign`, `rekor-cli` and `ec`.
+
+The complete sources for each cli tool are available in the source containers
+for the images that the binaries are extracted from.
```

Comment:
Can we point individuals to those repos? The images in the Dockerfile are what we use, but these should be pushed to registry.redhat.io as well ... and so we should be able to point readers there.

It is a bit awkward because it is very specific to our use cases.

Reply:
I've give it a try.

Are there some general purpose docs about how to find a source container for a particular image?